examples/bri-3/src/bri/identity/bpiAccounts/models/bpiAccount.ts

```diff
@@ -1,11 +1,11 @@
 export class BpiAccount {
```

Comment:
could you please execute this after all the PRs are merged, and then recreate your PR with latest changes? Otherwise we would need to put a lot of effort into merging.

Reply:
yes i will do this, in this PR after all pending now is merged